defaults/network-ents.go

```diff
@@ -38,10 +46,26 @@ func NetworkNoneEntitlement(profile *secProfile.Profile) (*secProfile.Profile, e
 
 	syscallsToBlock := []string{"socket", "socketpair", "setsockopt", "getsockopt", "getsockname", "getpeername",
 		"bind", "listen", "accept", "accept4", "connect", "shutdown", "recvfrom", "recvmsg", "sendto",
-		"sendmsg", "sendmmsg", "sethostname",
+		"sendmsg", "sendmmsg", "sethostname", "setdomainname",
 	}
 	profile.BlockSyscalls(syscallsToBlock...)
 
+	syscallsWithArgsToAllow := map[string][]specs.LinuxSeccompArg{
+		"socket": {
+			{
+				Index: 0,
+				Op:    specs.OpEqualTo,
+				Value: syscall.AF_UNIX,
+			},
+			{
+				Index: 0,
+				Op:    specs.OpEqualTo,
+				Value: syscall.AF_LOCAL,
```

Comment:
nit: it would be nice to make note of these args in the function docstring, since this isn't a full block of `socket`